helpers.json

```diff
@@ -962,6 +962,15 @@
             "canCarefulJump"
           ],
           "devNote": "canCarefulJump is used to represent leaving the enemy drops, which isn't a propper fit, but it is a tech with a similar difficulty."
+        },
+        {
+          "name": "h_pauseAbuseMinimalReserveRefill",
+          "requires": [
+            "canPauseAbuse",
+            {"resourceAtMost": [{"type": "RegularEnergy", "count": 1}]},
+            {"resourceConsumed": [{"type": "ReserveEnergy", "count": 1}]}
```

Comment:
Looks like Maddo's comment got lost to the ether. I think the main comment was relating to if this single energy drain is too precise for Expert and if it should be increased to 2. (in Extreme it should be fine at 1). 

I don't think it will really matter much, but these are the options that I could do ...
- fix it properly and add a leniency helper to this that adds 1 energy (probably overkill)
- keep it as is (maybe too tight on Expert)
- increase it to 2 (maybe too lenient on higher difficulties)

Reply:
Yeah another solution would be to add a tech for exact manual reserve energy control, and then add a leniency (say of 4) without that tech; that could be expressed as an `or` inside the helper. Because of the heat damage leniency, we're probably unlikely to see a scenario where this will matter in a significant way. So it could also be fine how it is. Adding a tech might be a good thing anyway though, just to have a place to document the technique to help players learn it.

Reply:
Added this. I dont think this helper should be used in heated rooms, but a heated variant could be made.